base/sort.jl

```diff
@@ -1221,6 +1530,8 @@ issignleft(o::ReverseOrdering, x::Floats) = lt(o, x, -zero(x))
 issignleft(o::Perm, i::Integer) = issignleft(o.order, o.data[i])
 
 function fpsort!(v::AbstractVector, a::Algorithm, o::Ordering)
+    length(v) < 10 && return sort!(v, first(axes(v,1)), last(axes(v,1)), SMALL_ALGORITHM, o)
```

Comment:
Add a comment about performance? Why wasn't this done before (seems not directly related to radix sort)?

Reply:
It's not directly related to radix, but at some point, radix introduced a bit of overhead in dispatch and this offset a regression. I'll add a comment abt performance.